app/src: properly log num_items value for addr_getNumItems

odeke-em opened the pull request:
This change fixes a bug in which "num_items 1\n" was unconditionally
and incorrectly being logged even when in expert mode, for which the
value was changed to 2; this incorrect condition would emit 2 different
logs with different values.

Fixes #59

cwgoes commented:
This can be replaced by the changes in https://github.com/tendermint/go-crypto/pull/85 (should work already); ideally we import go-crypto into the demo client.

odeke-em committed:
app/src: properly log num_items value for addr_getNumItems

This change fixes a bug in which "num_items 1\n" was unconditionally
and incorrectly being logged even when in expert mode, for which the
value was changed to 2; this incorrect condition would emit 2 different
logs with different values.

Fixes #59
